structuralcodes/materials/concrete/_concreteEC2_2004.py

```diff
@@ -167,16 +174,22 @@ def Ecm(self, value: float):
         """
         self._Ecm = abs(value)
 
-    def fcd(self, alpha_cc: float, gamma_c: float) -> float:
+    def fcd(self, alpha_cc: float) -> float:
```

Comment:
This is making me wonder if we can extend further the concept of partial factors.
For instance also alpha_cc has a recommended value of 1.0 or set by NAs between 0.8 and 1.0.

I think we could:

1. manage also alpha_cc in a way that can interact with NA or return the default value of 1.0

2. Alternatively we could simply se a default value to alpha_cc parameter in this function?

Reply:
Yes, I am thinking the same. But in my opinion, changing also alpha_cc and others according to the approach suggested here form gamma_c is a next step after we agree on the present suggestion 😃